nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/test/java/org/nuxeo/ecm/platform/tag/TestFacetedTagService.java

```diff
@@ -55,15 +53,11 @@
  *
  * @since 9.3
  */
-@Features(LogCaptureFeature.class)
```

Comment:
Then you can remove the unused import `import org.nuxeo.runtime.test.runner.Features;`